src/routes/api/articleRoutes.js

```diff
@@ -1,9 +1,26 @@
-import express from 'express'
+import express from 'express';
+import { ArticleController } from '../../controllers/articleController';
 
-const route = express.Router()
+const route = express.Router();
 
 route.get('/', (req, res, next) => {
-    res.status(200).json({ status: 200, message: "this will return all articles", data: "" })
-})
+  new ArticleController().getAllArticles(req, res, next);
+});
 
-export default route
\ No newline at end of file
+route.post('/', (req, res, next) => {
+  new ArticleController().createArticle(req, res, next);
+});
```

Comment:
Replace with `route.post("/", ArticleController.createArticle)` and do the same for other route